### DISCUSSION TITLE
how to avoid out of memory

### DISCUSSION THREAD

**Discussion by matteodelta** [projectdiscovery/httpx#675] [Q&A] [answered]
Using 1 GB VPS and scanning a big list of urls (suppose 1 million), the program often go out of memory. Is there any settings to tweak to avoid this?

**Comment by olearycrew** [comment, marked as answer]
You could split the file into multiple chunks or try increasing the delay / decreasing the timeout time

**Reply by brenocss** [reply]
Shouldn't this be automatic?
